CMakeLists.txt

```diff
@@ -24,27 +24,29 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.0.2)
 
 macro(CeleroSetDefaultCompilerOptions)
+	set_target_properties(${PROJECT_NAME} PROPERTIES
+		POSITION_INDEPENDENT_CODE "${CELERO_COMPILE_PIC}"
+	)
 	if(${CMAKE_CXX_COMPILER_ID} STREQUAL MSVC)
 		target_compile_options(${PROJECT_NAME} PRIVATE /D_VARIADIC_MAX=10 )
 		target_compile_options(${PROJECT_NAME} PRIVATE /D_CRT_SECURE_NO_WARNINGS)
 		target_compile_options(${PROJECT_NAME} PRIVATE /wd4251)
 		target_compile_options(${PROJECT_NAME} PRIVATE /MP)
 		target_compile_options(${PROJECT_NAME} PRIVATE /D_SCL_SECURE_NO_WARNINGS)
 		target_compile_options(${PROJECT_NAME} PRIVATE /permissive-)
-		target_compile_options(${PROJECT_NAME} PRIVATE /std:c++14)
 
         if(CELERO_TREAT_WARNINGS_AS_ERRORS)
             target_compile_options(${PROJECT_NAME} PRIVATE /WX)
 		endif()
+		set_target_properties(${PROJECT_NAME} PROPERTIES
+			ARCHIVE_OUTPUT_NAME "${PROJECT_NAME}.dll")
```

Comment:
`ARCHIVE_OUTPUT_NAME` controls filename of static and import lib. Anyway having extensions like `.dll.lib` for msvc should be avoided.

Reply:
I did this so the built artifacts for msvc (shared and lib) can be combined in one archive.
I don't know whether this is a hard requirement for celero.